src/website/frontend/styles/lib/colors.scss

```
@@ -0,0 +1,110 @@
+// **MAKE SURE TO FIRST SEARCH FOR YOUR COLOR (CTRL+F) BEFORE ADDING IT TO THE LIST**
+
+// BLUE
+$aq-blue-0: #135dff;
+
+$aq-blue-1: #ecf2ff;
```

Stylelint (17.6.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)